dc-cudami-editor/src/components/PagedIdentifiableList.jsx

```diff
@@ -232,106 +245,118 @@ class PagedIdentifiableList extends Component {
+                key={language}
+                language={language}
+                toggle={this.toggleLanguage}
               />
             ))}
-          </tbody>
-        </Table>
-        {identifiables.length > 0 && (
-          <ReactPaginate
-            activeClassName="active"
-            breakClassName="page-item"
-            breakLabel="&hellip;"
-            breakLinkClassName="page-link"
-            containerClassName="mt-2 pagination"
-            disabledClassName="disabled"
-            forcePage={pageNumber}
-            marginPagesDisplayed={1}
-            nextClassName="page-item"
-            nextLabel="&raquo;"
-            nextLinkClassName="page-link"
-            onPageChange={this.updatePage}
-            pageClassName="page-item"
-            pageCount={numberOfPages}
-            pageLinkClassName="page-link"
-            pageRangeDisplayed={5}
-            previousClassName="page-item"
-            previousLabel="&laquo;"
-            previousLinkClassName="page-link"
-          />
-        )}
+        </Nav>
+        <Card className="border-top-0">
+          <CardBody>
+            {this.state.identifiables.length > 0 && (
+              <ReactPaginate
+                activeClassName="active"
+                breakClassName="page-item"
+                breakLabel="&hellip;"
+                breakLinkClassName="page-link"
+                containerClassName="d-inline-flex mb-2 pagination"
+                disabledClassName="disabled"
+                forcePage={this.state.pageNumber}
+                marginPagesDisplayed={1}
+                nextClassName="page-item"
+                nextLabel="&raquo;"
+                nextLinkClassName="page-link"
+                onPageChange={this.updatePage}
+                pageClassName="page-item"
+                pageCount={this.state.numberOfPages}
+                pageLinkClassName="page-link"
+                pageRangeDisplayed={5}
+                previousClassName="page-item"
+                previousLabel="&laquo;"
+                previousLinkClassName="page-link"
+              />
+            )}
+            <span className="ml-2">
+              {t(`totalElements.${type}s`, {count: this.state.totalElements})}
+            </span>
+            <Table bordered className="mb-0" hover responsive size="sm" striped>
+              <thead>
+                <tr>
+                  <th className="text-right">
+                    <FaHashtag />
+                  </th>
+                  <th className="text-center">
+                    <FaImage />
+                  </th>
+                  <th>{t('label')}</th>
+                  <th>{t('identifiers')}</th>
+                  <th className="text-center">{t('lastModified')}</th>
+                  <th className="text-center">{t('actions')}</th>
+                </tr>
+              </thead>
+              <tbody>
+                {identifiables.map((identifiable, index) => (
+                  <IdentifiableListItem
```

Comment:
Hat jetzt mit diesem PR nich direkt was zu tun, aber warum nimmt denn diese Komponente lauter einzelne Felder von `Identifiable` als Prop und nicht einfach statdessen ein einzelnes`identifiable` prop? Das Interface so wie's jetzt ist ist ja schon wirklich extrem ausufernd.

Reply:
Werde ich in einem separaten PR korrigieren. Danke auf alle Fälle für die Anmerkung!